tests/integration/run.bats

```diff
@@ -38,6 +38,7 @@ function docker_node_nabla_run() {
 }
 
 function runnc_run() {
```

Comment:
Let's use this existing function so that we try not to dup code, with a few changes:
- Let's call the option `daemon` instead, so add the `daemon` flag to use in daemon mode (which will not block).
- Invert the if statement since we don't do anything. (also, `continue` is not the right keyword for `if/else` in bash i believe), I'm getting errors on my build from that line.